pkg/subctl/operator/deploy/ensure.go

```diff
@@ -0,0 +1,75 @@
+package deploy
+
+import (
+	"fmt"
+
+	submariner "github.com/submariner-io/submariner-operator/pkg/apis/submariner/v1alpha1"
+	submarinerclientset "github.com/submariner-io/submariner-operator/pkg/client/clientset/versioned"
+	"github.com/submariner-io/submariner-operator/pkg/subctl/datafile"
+
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/rest"
+)
+
+func Ensure(config *rest.Config, remoteNamespace string, submarinerNamespace string,
```

Comment:
Nit: "Ensure" is not a super-clear function name to me.

Reply:
I accept suggestions, my vocabulary is a bit short in that area.
https://www.thesaurus.com/browse/ensure

Provide? :? 

Anyway, that's not part of @skitt patch, github is horrible with chaining patches. I'll send another PR to handle that which is merged.

Thanks daniel

Reply:
For me the confusion is that I'm left with a question like "ensure what does what?". IMHO a more verbose function name, maybe adding a subject and/or verb, would be worth it for the extra clarity. Again, it's just a small nit and thanks!